Docs/Reports/Deliverables/D5.1/main.tex

```diff
@@ -252,7 +252,35 @@ \subsubsection{Neutrons}
 \subsubsection{Photons}
 \marginpar{TODO: Aljosa }
 \subsection{Molecular Dynamics data}
-\marginpar{TODO: Juncheng }
+Molecular dynamics (MD) is a long--established computer simulation method to analyze atom/molecular movements by solving Newton's
+equations of motion assuming a given potential energy function. It is broadly used in statistical physics, physical chemistry,
+structural biology, material science, and crystallography. As many researches in above disciplines are largely dependent
+on advanced light sources like synchrotron and free--electron lasers, MD simulation becomes an important tool to predict the 
+results of the experiments conducted in these large light source facilities.
+
+With MD simulation, the structural evolution of the experimental sample can be modeled. Depending on the computing power, the time scale 
+of the model can range from fs to ns, and the length scale can be up to several hundreds nms. The fact that the scales are 
+just right close the the experimental scales and the atomic-level simulation solution make it possible to calculate 
+scattering signals from the MD atomic configuration which is comparable with the experimental sample. This kind of simulations has already
+been used in the researches of deformation under high pressure\cite{kimminau2008jpcm,tang2017jap,tang2018jsr,zhang2019jsr}, 
+crystallization \cite{e2015}, and crystallography \cite{coleman2013msmse} to predict diffraction patterns for experimental design
+and interpretation.
+
+Not only the event that people want to explore in experiment, but also the interaction between sample and incident X--ray can be considered
```

Comment:
~~Not only the event that people want to explore in experiment, but also the interaction between sample and incident X--ray can be considered
in MD simulations.~~